### DISCUSSION TITLE
Unable to delete lobe-chat when removing from docker desktop

### DISCUSSION THREAD

**Discussion by Couch9388** [lobehub/lobehub#2141] [General Question] [answered]
### 💻 Operating System

macOS

### 📦 Environment

Docker

### 🌐 Browser

Chrome

### 🐛 Bug Description

I've got a docker desktop setup with lobe-chat, but when I am trying to delete the lobe-chat docker container, and try to pull again from docker desktop, it stores the previous setup that I have (with the same ports and configuration), which makes it impossible to delete the applications to start from scratch.

### 🚦 Expected Behavior

When I delete the container from docker desktop, all the previous configuration and setup should be deleted from local storage. Which make it easier to start from scratch.

**Comment by arvinxx** [comment, marked as answer]
you can just delete the data in the indexedDB 
<img width="859" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/caa1cb00-e3f4-4ab2-a801-096398b89363">

**Reply by Couch9388** [reply]
I see, is there a roadmap for making a visual indicator in the application itself that said something like "DELETE DATABASE, ACTION IRRETRIEVABLE". Just bring it out there for people that are not tech savvy.
明白了，不知是否会在下次的开发中，将这个作为一个功能。可以放在关于的下面，写着“删除”，里面会有一行字，但是会有一个红色的按钮，“删除数据库，无法撤回”？只是会觉得这样对于一些电脑小白来说，可行性比较大。

**Reply by arvinxx** [reply]
There is no possible to do that. I think the user who can use docker isn't a tech savvy

Or you can go to setting and you can clear all data.
